ext/reflection/tests/bug74351.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Bug #74351 (Wrong reflection on variadic array functions)
+--FILE--
+<?php
+$functions = [
+    'array_diff',
```

Comment:
Yes, that's really just because userland interfaces should be the authoritative approach